CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.1] - 2019-06-03
+
+- Added `Istanbul` HF candidate [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108)
+  (`DRAFT`) updated `alt_bn128` precompile gas costs (see `hardforks/istanbul.json`)
```

Comment:
`Updated ...` can be a separate item

Reply:
That was actually meant as one sentence "added the updated precompile gas costs being a Istanbul HF candidate", can we go with the current version eventually? 😎🙂

Reply:
Ah misread that